lib/displayHelpers.ts

```diff
@@ -31,9 +35,6 @@ const thinSpace = "\u202F";
  * @return {string} The abbreviated string.
  */
 const printableCoin = (coin: Coin, chainInfo: ChainInfo) => {
-  // null, undefined and this sort of things
-  if (!coin) return "–";
```

Comment:
Good change. Let's turn this into an `assert(coin, "Coin must not be falsy");` to get informed when something strange is happening.

Reply:
I can add that assert no problem, but what needs to be fixed is the `msg.value.amount` argument in `<TransactionInfo />` being `any`. If the function is safely called without `any`s `coin` should never be falsy

Reply:
Jupp, agreed. I just don't fully trust this app which was developed in JavaScript originally and stores and reads unchecked data from a database. I'm fine either way.